sangbok/main.xml

```diff
@@ -4592,4 +4592,54 @@
 			hur innerligt jag älskar dig!
 		</p>
 	</song>
+	
+	<song name="I Californien"
+		  category="Esoterica">
```

Comment:
Isn't there any additional info available about this song, like the author? Especially the melody would help...